armi/reactor/tests/test_assemblies.py

```diff
@@ -278,6 +278,36 @@ def setUp(self):
 
         self.assembly.calculateZCoords()
 
+    def test_notesParameter(self):
+        self.assertEqual(self.assembly.p.notes, "")
+
+        with self.assertRaises(ValueError):
+            # try to assign a non-string
+            self.assembly.p.notes = 1
+
+        note = "This is a short, acceptable not about the assembly"
+        self.assembly.p.notes = note
+        self.assertEqual(self.assembly.p.notes, note)
+
+        tooLongNote = r"""
+This is a note that is too long. asdfasdfasdfadslfkasjdflkasdjflaksjdf;adkfasfsd
```

Comment:
LOL

I mean, you know you can do `"abcde"*201`, right?

Reply:
riiiight...

It was fun typing that out though.

Will update!